index.html

```diff
@@ -0,0 +1,163 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Mark.O | Capstone One</title>
+    <script src="script.js" defer></script>
+</head>
+
+<body>
+    <header>
+        <nav class="mobile-nav flex">
+            <img src="./cap-detials/cross-con.png" class="cross-icon" alt="crossIcon">
+            <ul class="flex">
+                <li><a href="./index.html" class="navLink">Home</a></li>
+                <li><a href="./about.html" class="navLink">About</a></li>
+                <li><a href="#program" class="navLink">Program</a></li>
+                <li><a href="#" class="navLink">Join</a></li>
+                <li><a href="#nominees" class="navLink">nominees</a></li>
+                <li><a href="#partners" class="navLink">Partners</a></li>
+            </ul>
+        </nav>
```

Comment:
- [OPTIONAL] What about preventing the mobile nav page from scrolling when the navbar is open? You can use a CSS property `over-flow: hidden;` to do that, which will improve your mobile menu design.💯 😄